subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
                     <li><a href="https://guides.gradle.org/migrating-from-maven/">Migrating From Maven</a></li>
                 </ul>
             </li>
-            <li id="tutorials"><a href="https://guides.gradle.org/#getting-started">Getting Started Guides</a></li>
-            <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
+            <li id="tutorials"><a href="https://guides.gradle.org/#getting-started">Tutorials</a></li>
         </ul>
 
         <h3 id="reference">Reference</h3>
         <ul>
-            <li><a href="https://gradle.github.io/kotlin-dsl-docs/api/" target="_blank">Kotlin DSL Reference</a></li>
-            <li><a href="/dsl/">Groovy DSL Reference</a></li>
-            <li><a href="/javadoc/">Gradle API Javadoc</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#gradle-api" aria-expanded="false" aria-controls="gradle-api">Gradle API</a>
+                <ul id="gradle-api">
+                    <li><a href="/javadoc/">Javadoc</a></li>
+                    <li><a href="/dsl/">Groovy DSL</a></li>
+                    <li><a href="https://gradle.github.io/kotlin-dsl-docs/api/" target="_blank">Kotlin DSL</a></li>
+                </ul>
+            </li>
             <li><a href="/userguide/command_line_interface.html">Command-Line Interface</a></li>
             <li><a href="/userguide/plugin_reference.html">Core Plugins</a></li>
+            <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
             <li><a href="https://docs.gradle.com/build-scan-plugin">Build Scan Plugin User Manual</a></li>
         </ul>
 
-        <h3 id="getting-started">Managing Gradle Projects</h3>
+        <h3 id="getting-started">Running Gradle Builds</h3>
         <ul>
-            <li><a href="https://guides.gradle.org/creating-new-gradle-builds/">Creating New Gradle Builds</a></li>
-            <li><a href="https://guides.gradle.org/creating-build-scans/">Inspecting Gradle Builds</a></li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#organizing-gradle-projects" aria-expanded="false" aria-controls="organizing-gradle-projects">Organizing Project Structure</a>
-                <ul id="organizing-gradle-projects">
-                    <li><a href="/userguide/organizing_gradle_projects.html">Organizing Projects Best Practices</a></li>
-                    <li><a href="/userguide/directory_layout.html">Directory Layout</a></li>
-                    <li><a href="/userguide/composite_builds.html">Integrating Separate Gradle Builds (Composite Builds)</a></li>
-                    <li><a href="/userguide/intro_multi_project_builds.html">Multi-Project Builds</a></li>
+            <li>Using the Command Line (TBD)</li>
+            <li>Using Gradle From an IDE (TBD)</li>
```

Comment:
These items represent what I perceive as glaring holes in the documentation. The "Command-Line Interface" chapter is more of a reference than a gentle introduction to running Gradle builds from the command line. And we desperately need something for IDEs.

Reply:
I've put them in as a guide for what I think should be included in this section in the future. I don't propose to keep them for the Gradle 5.0 release unless I suddenly become super productive and knock out the corresponding chapters in short order!

Reply:
I agree these are both needed. I've filed issues on the right GitHub board and put them in the Backlog.

I think "Gradle Concepts" has to be my next `#1` unwritten thing as you may have seen elsewhere.